mozilla-release/browser/components/preferences/in-content/experiments.js

```diff
@@ -0,0 +1,19 @@
+var { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
+
+var gExperimentsPane = {
+  _pane: null,
+
+  init: function ()
+  {
+    this._pane = document.getElementById("paneExperiments");
```

Comment:
@sammacbeth Good job! The only thing is that I am not sure we need this._pane here. Since the element with id="paneExperiments" does not exist (at least I couldn't find it). Probably you have followed the same pattern we have for **privacy.js** / **connect.js** files. I have checked privacy section and this._pane is null there as well (we need to remove this._pane there in the future also).

Reply:
We need to do that for `connect.js` too - `_pane` looks unnecessary.

Reply:
Done in https://github.com/cliqz-oss/browser-f/pull/1422